pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -219,6 +249,14 @@ def submissions(self):
                                        creation_time=self.object.commented_on)
                               .order_by("id"))
 
+    @cached_property
+    def comments(self):
+        return dict(
+            get_model().objects
+                       .filter(suggestions__unit=self.object)
+                       .values_list("suggestions__id", "comment")
+        )
```

Comment:
It seems like this doesnt work on postgres as it cant join the gfkey which is a string to the object_pk.

Why did you not want to do `object__pk__in=suggestions` ?